### DISCUSSION TITLE
[3.3.5] Holy Resistances

### DISCUSSION THREAD

**Discussion by Jinnaix** [TrinityCore/TrinityCore#26527] [General]
Greetings everyone.

I suggest to get rid of all creature holy resistances in the database.
As far as i could research this, natural holy resistance only exists (or should only exist) as a formula that adds resistance by level difference in PVE.

I am aware of the beast lore ability and that it could show resistances back then. But we still have resistances in the database from non beast creatures like elementals and other stuff.

Some references:

Blizzards Editor back then that didn't even had the option to give holy resistances to creatures:
<details>
  <summary>Click to expand!</summary>

![image](https://user-images.githubusercontent.com/37972361/118252505-21888580-b4a9-11eb-8e83-08465a5c9cb6.png)

</details>

Written stuff to read about this:

[Holy Resistances in Wikis](https://wowwiki-archive.fandom.com/wiki/Holy_resistance)
[Holy Resistances in Wikis 2](https://wowpedia.fandom.com/wiki/Holy_resistance)
[Some Battle.Net Forum discussions](https://us.forums.blizzard.com/en/wow/t/holy-resistance-correct-my-memory/305509)
[And more](https://us.forums.blizzard.com/en/wow/t/holy-resistance/181447)
[Reddit is also in](https://www.reddit.com/r/classicwow/comments/c3q6d1/holy_spell_resist_for_mobsraid_bosses/)
[with some topics about this](https://www.reddit.com/r/classicwow/comments/fy17ri/can_holy_spells_be_resisted/)
[And so is azerothcore](https://github.com/azerothcore/azerothcore-wotlk/issues/5453)

```sql
SELECT * FROM `creature_template_resistance` WHERE School = '1';
```

<details>
  <summary>Click to expand!</summary>

creature_template_resistance
---
| CreatureID | School | Resistance | VerifiedBuild | 
| ---: | ---: | ---: | ---: | 
| 157 | 1 | 10 | 12340 | 
| 329 | 1 | 260 | 12340 | 
| 521 | 1 | 80 | 12340 | 
| 764 | 1 | 39 | 12340 | 
| 765 | 1 | 40 | 12340 | 
| 766 | 1 | 41 | 12340 | 
| 1501 | 1 | 1 | 12340 | 
| 1502 | 1 | 1 | 12340 | 
| 1812 | 1 | 55 | 12340 | 
| 1813 | 1 | 114 | 12340 | 
| 2166 | 1 | 18 | 12340 | 
| 2359 | 1 | 34 | 12340 | 
| 2745 | 1 | 42 | 12340 | 
| 2761 | 1 | 39 | 12340 | 
| 2776 | 1 | 40 | 12340 | 
| 2794 | 1 | 190 | 12340 | 
| 2887 | 1 | 41 | 12340 | 
| 2919 | 1 | 45 | 12340 | 
| 3516 | 1 | 504 | 12340 | 
| 3630 | 1 | 2 | 12340 | 
| 3631 | 1 | 2 | 12340 | 
| 3950 | 1 | 1 | 12340 | 
| 4028 | 1 | 26 | 12340 | 
| 4029 | 1 | 28 | 12340 | 
| 4030 | 1 | 240 | 12340 | 
| 4034 | 1 | 25 | 12340 | 
| 4423 | 1 | 120 | 12340 | 
| 4535 | 1 | 20 | 12340 | 
| 4661 | 1 | 30 | 12340 | 
| 5056 | 1 | 2 | 12340 | 
| 5291 | 1 | 2 | 12340 | 
| 5317 | 1 | 244 | 12340 | 
| 5320 | 1 | 248 | 12340 | 
| 5349 | 1 | 85 | 12340 | 
| 5356 | 1 | 100 | 12340 | 
| 5481 | 1 | 96 | 12340 | 
| 5485 | 1 | 50 | 12340 | 
| 5756 | 1 | 2 | 12340 | 
| 5761 | 1 | 100 | 12340 | 
| 5806 | 1 | 23 | 12340 | 
| 5834 | 1 | 55 | 12340 | 
| 5853 | 1 | 46 | 12340 | 
| 5881 | 1 | 45 | 12340 | 
| 5890 | 1 | 1 | 12340 | 
| 6074 | 1 | 23 | 12340 | 
| 6129 | 1 | 216 | 12340 | 
| 6220 | 1 | 145 | 12340 | 
| 6239 | 1 | 40 | 12340 | 
| 6492 | 1 | 17 | 12340 | 
| 6498 | 1 | 200 | 12340 | 
| 6499 | 1 | 200 | 12340 | 
| 6500 | 1 | 200 | 12340 | 
| 6550 | 1 | 40 | 12340 | 
| 7031 | 1 | 52 | 12340 | 
| 7032 | 1 | 112 | 12340 | 
| 7104 | 1 | 560 | 12340 | 
| 7132 | 1 | 54 | 12340 | 
| 7226 | 1 | 46 | 12340 | 
| 7266 | 1 | 32 | 12340 | 
| 7768 | 1 | 40 | 12340 | 
| 7780 | 1 | 40 | 12340 | 
| 8211 | 1 | 90 | 12340 | 
| 8213 | 1 | 51 | 12340 | 
| 8281 | 1 | 294 | 12340 | 
| 8303 | 1 | 30 | 12340 | 
| 8400 | 1 | 104 | 12340 | 
| 8441 | 1 | 48 | 12340 | 
| 8667 | 1 | 44 | 12340 | 
| 8905 | 1 | 108 | 12340 | 
| 8906 | 1 | 110 | 12340 | 
| 8911 | 1 | 55 | 12340 | 
| 8981 | 1 | 110 | 12340 | 
| 8982 | 1 | 290 | 12340 | 
| 9017 | 1 | 55 | 12340 | 
| 9025 | 1 | 510 | 12340 | 
| 9026 | 1 | 104 | 12340 | 
| 9377 | 1 | 34 | 12340 | 
| 9397 | 1 | 48 | 12340 | 
| 9453 | 1 | 54 | 12340 | 
| 9598 | 1 | 112 | 12340 | 
| 10077 | 1 | 100 | 12340 | 
| 10442 | 1 | 55 | 12340 | 
| 10641 | 1 | 250 | 12340 | 
| 10882 | 1 | 2 | 12340 | 
| 10955 | 1 | 60 | 12340 | 
| 11357 | 1 | 2 | 12340 | 
| 11440 | 1 | 54 | 12340 | 
| 11441 | 1 | 58 | 12340 | 
| 11442 | 1 | 54 | 12340 | 
| 11443 | 1 | 53 | 12340 | 
| 11444 | 1 | 58 | 12340 | 
| 11480 | 1 | 120 | 12340 | 
| 11483 | 1 | 290 | 12340 | 
| 11484 | 1 | 120 | 12340 | 
| 11489 | 1 | 300 | 12340 | 
| 11491 | 1 | 290 | 12340 | 
| 11576 | 1 | 34 | 12340 | 
| 11577 | 1 | 37 | 12340 | 
| 11578 | 1 | 34 | 12340 | 
| 11669 | 1 | 122 | 12340 | 
| 11744 | 1 | 56 | 12340 | 
| 11745 | 1 | 116 | 12340 | 
| 11783 | 1 | 92 | 12340 | 
| 11784 | 1 | 240 | 12340 | 
| 11862 | 1 | 25 | 12340 | 
| 11871 | 1 | 2 | 12340 | 
| 12431 | 1 | 70 | 12340 | 
| 13278 | 1 | 300 | 12340 | 
| 13279 | 1 | 56 | 12340 | 
| 13282 | 1 | 96 | 12340 | 
| 13285 | 1 | 58 | 12340 | 
| 13322 | 1 | 290 | 12340 | 
| 13456 | 1 | 45 | 12340 | 
| 13656 | 1 | 100 | 12340 | 
| 13696 | 1 | 92 | 12340 | 
| 13736 | 1 | 46 | 12340 | 
| 14025 | 1 | 180 | 12340 | 
| 14232 | 1 | 60 | 12340 | 
| 14234 | 1 | 2 | 12340 | 
| 14362 | 1 | 50 | 12340 | 
| 14399 | 1 | 120 | 12340 | 
| 14400 | 1 | 60 | 12340 | 
| 14454 | 1 | 600 | 12340 | 
| 14458 | 1 | 57 | 12340 | 
| 14460 | 1 | 55 | 12340 | 
| 14461 | 1 | 116 | 12340 | 
| 14464 | 1 | 58 | 12340 | 
| 15339 | 1 | 1000 | 12340 | 
| 19668 | 1 | 350 | 12340 | 

</details>

```sql
DELETE FROM `creature_template_resistance` WHERE School = '1';
```

**Comment by offl** [comment]
Yep, I remember I did the same long time ago
Should not be confused with immune masks because I use full immune mask to grant full immunity to spells
